### PR TITLE
Material Icons update and change Reddit to Docker

### PR DESCRIPTION
**Pull request by ellnic**

### CODE REVIEW

File: static/css/themes/Light.css [hunk 1]
```diff
@@ -3,4 +3,8015 @@
...
...wrap:wrap;padding:.75rem 1rem;margin-bottom:1rem;list-style:none;background-color:#e9ecef}.breadcrumb-item+.breadcrumb-item::before{display:inline-block;padding-right:.5rem;padding-left:.5rem;color:#868e96;content:"/"}.breadcrumb-item+.breadcrumb-item:hover::before{text-decoration:underline}.breadcrumb-item+.breadcrumb-item:hover::before{text-decoration:none}.breadcrumb-item.active{color:#868e96}.pagination{display:flex;padding-left:0;list-style:none}.page-link{position:relative;display:block;padding:.5rem .75rem;margin-left:-1px;line-height:1.25;color:#2196F3;background-color:#fff;border:1px solid #dee2e6}.page-link:hover{color:#0a6ebd;text-decoration:none;background-color:#e9ecef;border-color:#dee2e6}.page-link:focus{z-index:2;outline:0;box-shadow:0 0 0 .2rem rgba(33,150,243,0.25)}.page-link:not(:disabled):not(.disabled){cursor:pointer}.page-item:first-child .page-link{margin-left:0}.page-item.active .page-link{z-index:1;color:#fff;background-color:#2196F3;border-color:#2196F3}.page-item.disabled .page-link{color:#868e96;pointer-events:none;cursor:auto;background-color:#fff;border-color:#dee2e6}.pagination-lg .page-link{padding:.75rem 1.5rem;font-size:1.17188rem;line-height:1.5}.
```
Comment (barbequesauce) [inline static/css/themes/Light.css]: Codacy Issue found: [Use of !important](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4602655)

**Comment by barbequesauce** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 50
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/Watcher3/Watcher3/pullRequest?prid=4602655&bid=15446073)

File: static/css/themes/Light.css (hunk 1 above)
Comment (barbequesauce) [inline static/css/themes/Light.css]: Codacy Issue found: [Expected RBRACE at line 32, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4602655)

Comment (barbequesauce) [inline static/css/themes/Light.css]: Codacy Issue found: [Expected RBRACE at line 25, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4602655)

Comment (barbequesauce) [inline static/css/themes/Light.css]: Codacy Issue found: [Expected RBRACE at line 27, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4602655)

Comment (barbequesauce) [inline static/css/themes/Light.css]: Codacy Issue found: [Expected RBRACE at line 35, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4602655)

Comment (barbequesauce) [inline static/css/themes/Light.css]: Codacy Issue found: [Expected RBRACE at line 15, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4602655)

Comment (barbequesauce) [inline static/css/themes/Light.css]: Codacy Issue found: [Expected RBRACE at line 31, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4602655)

Comment (barbequesauce) [inline static/css/themes/Light.css]: Codacy Issue found: [Use of !important](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4602655)

Comment (barbequesauce) [inline static/css/themes/Light.css]: Codacy Issue found: [Expected RBRACE at line 14, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4602655)

Comment (barbequesauce) [inline static/css/themes/Light.css]: Codacy Issue found: [Expected COLON at line 4101, col 1.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4602655)

Comment (barbequesauce) [inline static/css/themes/Light.css]: Codacy Issue found: [Expected RBRACE at line 12, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4602655)

Comment (barbequesauce) [inline static/css/themes/Light.css]: Codacy Issue found: [Use of !important](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4602655)

Comment (barbequesauce) [inline static/css/themes/Light.css]: Codacy Issue found: [Use of !important](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4602655)

Comment (barbequesauce) [inline static/css/themes/Light.css]: Codacy Issue found: [Use of !important](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4602655)

Comment (barbequesauce) [inline static/css/themes/Light.css]: Codacy Issue found: [Expected RBRACE at line 29, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4602655)

Comment (barbequesauce) [inline static/css/themes/Light.css]: Codacy Issue found: [Use of !important](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4602655)

Comment (barbequesauce) [inline static/css/themes/Light.css]: Codacy Issue found: [Expected RBRACE at line 30, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4602655)

Comment (barbequesauce) [inline static/css/themes/Light.css]: Codacy Issue found: [Expected RBRACE at line 21, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4602655)

Comment (barbequesauce) [inline static/css/themes/Light.css]: Codacy Issue found: [Expected RBRACE at line 9, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4602655)

Comment (barbequesauce) [inline static/css/themes/Light.css]: Codacy Issue found: [Expected RBRACE at line 36, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4602655)

Comment (barbequesauce) [inline static/css/themes/Light.css]: Codacy Issue found: [Expected RBRACE at line 11, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4602655)

Comment (barbequesauce) [inline static/css/themes/Light.css]: Codacy Issue found: [Use of !important](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4602655)

Comment (barbequesauce) [inline static/css/themes/Light.css]: Codacy Issue found: [Expected RBRACE at line 26, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4602655)

Comment (barbequesauce) [inline static/css/themes/Light.css]: Codacy Issue found: [Expected RBRACE at line 16, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4602655)

Comment (barbequesauce) [inline static/css/themes/Light.css]: Codacy Issue found: [Expected RBRACE at line 28, col 2.](https://app.codacy.com/app/saucebarbeque/Watcher3/pullRequest?prid=4602655)

**Comment by ellnic** [comment]
Sorry this is my fault. I meant to submit this to my changes branch, not master. I'll close it and resubmit.

**Pull request closed without merging**